show/bgp_frr_v4.py

```diff
@@ -0,0 +1,131 @@
+import click
+from show.main import *
+
+
+################################################################################
+#
+# 'show bgp' cli stanza
+#
+################################################################################
+
+
+@cli.group(cls=AliasedGroup, default_if_no_args=False)
+def bgp():
+    """Show IPv4 BGP (Border Gateway Protocol) information"""
+    pass
+
+
+# 'summary' subcommand ("show bgp summary")
+@bgp.command()
+def summary():
+    """Show summarized information of IPv4 BGP state"""
+    print "FRR"
+    run_command('sudo vtysh -c "show bgp summary"')
+
+
+# 'neighbors' subgroup ("show bgp neighbors ...")
+@bgp.group(cls=AliasedGroup, default_if_no_args=True)
+def neighbors():
+    """Show BGP neighbors"""
+    pass
+
+
+# 'neighbors' subcommand ("show bgp neighbors")
+@neighbors.command(default=True)
+@click.argument('ipaddress', required=False)
+def default(ipaddress):
+    """Show BGP neighbors"""
+
+    if ipaddress is not None:
+        command = 'sudo vtysh -c "show bgp neighbor {} "'.format(ipaddress)
+        run_command(command)
+    else:
+        run_command('sudo vtysh -c "show bgp neighbor"')
+
+
+# 'advertised-routes' subcommand ("show bgp neighbors advertised-routes <nbr>")
+@neighbors.command('advertised-routes')
+@click.argument('ipaddress')
+def advertised_routes(ipaddress):
+    command = 'sudo vtysh -c "show bgp ipv4 neighbor {} advertised-routes"'.format(ipaddress)
+    run_command(command)
+
+
+# 'received-routes' subcommand ("show bgp neighbors received-routes <nbr>")
+@neighbors.command('received-routes')
+@click.argument('ipaddress')
+def received_routes(ipaddress):
+    command = 'sudo vtysh -c "show bgp ipv4 neighbor {} received-routes"'.format(ipaddress)
```

Comment:
this is not supported, because we did not enable route cache in quagga. use show bgp ipv4 neighbor {} routes instead.

Reply:
Good catch. Will remove this command for the time being.